File: week06/week06-4.cpp
int maxFrequencyElements(int* nums, int numsSize) {
    int a[101] = {};
    int best = 0; 
    for(int i=0;i<numsSize; i++){
        int now=nums[i];
        a[now]++;
        if(a[now]>best) best=a[now];
    }
    int ans=0;
    for(int i=1;i<=100;i++){
        if(a[i]==best) ans += a[i];
    }
    return ans;
}

File: week06/week06-4.py
class Solution:
    def maxFrequencyElements(self, nums: List[int]) -> int:
        a=[0]*101
        best=0
        for now in nums:
            a[now]+=1
            if a[now]>best: best=a[now]

        ans=0
        for i in range(1,101):
            if a[i]==best: ans+=a[i]
        return ans